torch/_inductor/ir.py

```diff
@@ -93,6 +93,8 @@ def _check_tensorbox(node):
             (
```

Comment:
This function is actually about what valid RETURN types from Inductor are, this isn't just IR